### DISCUSSION TITLE
alex_16gpu, r50_8gpu, etc in model zoo

### DISCUSSION THREAD

**Discussion by GOBish** [STVIR/pysot#379] [General]
I see that the experiments folder has configs for 
siamrpn_alex_dwxcorr_16gpu
siamrpn_r50_l234_dwxcorr_8gpu
siamrpn_r50_l234_dwxcorr_16gpu
siamrpn_r50_l234_dwxcorr_otb_8gpu

but these models are not in the model zoo. Is there a place we can find them?

Thanks.

**Comment by ZhiyuanChen** [comment]
Why do you need a pretrain weight for a config that the only difference is the number of GPU used for training?
